pupil-spa/src/app/login-failure/login-failure.component.html

```diff
@@ -0,0 +1,45 @@
+<app-page-modifications>
+  <app-header class="full-width"></app-header>
+  <main id="content" class="js-content" role="main">
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <header class="page-header">
+          <h1 class="aa-title-size" tabindex="0">Sign in error</h1>
+        </header>
+        <br />
+        <div class="notice">
+          <strong class="visually-hidden">Warning</strong>
+          <app-svg-warning></app-svg-warning>
```

Comment:
Is this tag doing anything?

Reply:
@jon-shipley that's the warning svg icon